soteria/lib/soteria_std/seq.ml

```diff
@@ -12,6 +14,7 @@ let self_cross_product seq =
   in
   aux seq
 
+(** Create a sequence of values generated by a function up to a given bound. *)
```

Comment:
Maybe emphasise that the difference with Stdlib.Seq.init is that the input parameter is a Z.t instead of an int